integration/src/integration/java/pl/allegro/tech/hermes/integration/env/ConsumersStarter.java

```diff
@@ -50,6 +50,8 @@ public void start() throws Exception {
         overrideProperty(CONSUMER_USE_TOPIC_MESSAGE_SIZE, true);
         overrideProperty("consumer.ssl.keystoreSource", "provided");
         overrideProperty("consumer.ssl.truststoreSource", "provided");
+        overrideProperty("consumer.commit.offset.queuesInflightDrainFullEnabled", true);
```

Comment:
the name of these properties should be as public static inside `pl.allegro.tech.hermes.common.config.Configs`